docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -264,421 +273,403 @@
-			<p>Comming soon</p>
+			<p><strong>Preference</strong> [In progress]</p>
+			<p>This sample application demonstrates how to store and retrieve an application specific data and preference using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Preference.html" target="_blank">Tizen.Applications.Preference C# API</a>.</p>
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="267" src="media/m32speechtotext.png" width="150" /></td>
+			<td><img alt="" height="267" src="media/m32speechtotext.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/SpeechToText" target="_blank"><strong>SpeechToText</strong></a></p>
-			<p>SpeechToText is a sample application which demonstrates how to record and recognize sound data using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Uix.Stt.html" target="_blank">Tizen STT API</a>.</p>
+			<p>This sample application demonstrates how to record and recognize sound data using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Uix.Stt.html" target="_blank">Tizen.Uix.Stt C# API</a>.</p>
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="267" src="media/m33smartcard.png" width="150" /></td>
+			<td><img alt="" height="267" src="media/m33smartcard.png" width="150"/></td>
 			<td>
-			<p><strong>Smartcard</strong></p>
-			<p>The sample application demonstrates how to work with Smartcard API in order to send command to Secure Element (SE).</p>
-			<p>Comming soon</p>
+			<p><strong>Smartcard</strong> [In progress]</p>
+			<p>This sample application demonstrates how to work with Smartcard API in order to send command to Secure Element (SE).</p>
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="267" src="media/m34gallery.png" width="150" /></td>
+			<td><img alt="" height="267" src="media/m34gallery.png" width="150"/></td>
 			<td>
-			<p><strong>Gallery</strong></p>
-			<p>Gallery is a sample application that demonstrates how to create image viewer using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Content.MediaContent.html" target="_blank">Tizen Content API</a>.</p>
-			<p>Comming soon</p>
+			<p><strong>Gallery</strong> [In progress]</p>
+			<p>This sample application demonstrates how to create image viewer using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Content.MediaContent.html" target="_blank">Tizen.Content.MediaContent C# API</a>.</p>
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="267" src="media/m35nfc.png" width="150" /></td>
+			<td><img alt="" height="267" src="media/m35nfc.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NFCSampleApp" target="_blank"><strong>NFC</strong></a></p>
```

Comment:
This link is not working.

Reply:
The link will be available soon. Jihun Jung (jh8801.jung) is fixing it.